src/stores/EntryStore.ts

```diff
@@ -89,8 +89,16 @@ export default class EntryStore extends EventEmitter {
 
   public loadQueryString(qs:string) {
     let params = querystring.parse(location.search);
-    for (let key in params) {
-      this.tracks[key] = params[key].split(/,/);
+    let pkeys = Object.keys(params);
+    let keys = Object.keys(this.tracks);
+    for (let k of pkeys) {
+      if (keys.indexOf(k) === -1) {
+        keys.push(k);
+      }
+    }
+    for (let key of keys) {
+      let track = params[key] || '';
```

Comment:
`params[key]` can only be string or undefined, so it's not necessary to give it an empty string.